src/express/views/components/Header/Buttons/SignUpButton/SignUpButton.tsx

```diff
@@ -1,8 +1,5 @@
 import {ClientRoutes} from "../../../../../../constants-es6";
 import React from "react";
+import {DefaultButton} from "@fluentui/react";
 
-export const SignUpButton = () => (
-  <a className="header__registration" href={ClientRoutes.REGISTRATION}>
-    Регистрация
-  </a>
-);
+export const SignUpButton = () => <DefaultButton href={ClientRoutes.REGISTRATION}>Регистрация</DefaultButton>;
```

Comment:
```suggestion
export const SignUpButton = () => <DefaultButton href={ClientRoute.REGISTRATION}>Регистрация</DefaultButton>;
```
 Енамы должны быть в единственном числе

Reply:
Done